packages/protvista-utils/src/TrackHighlighter.js

```diff
@@ -89,14 +96,14 @@ export default class TrackHighlighter {
       .enter()
       .append("rect")
       .style("opacity", 0.5)
```

Comment:
I wonder if we still need this line with the alpha defined in the hex color. Might need to drop the value a little in the hex default value definition to match the current color (or slightly darker, as it's sometimes difficult to see for variants)